src/services/Http.js

```diff
@@ -274,7 +289,8 @@ export class Http {
       axios({
         method: method,
         // url: `https://${version}.openx.solar/${path}`,
-        url: `http://3.208.3.207:8081/${path}`,
+        // url: `https://demoapi.openx.solar:8081/${path}`,
+        url: `http://localhost:8081/${path}`,
```

Comment:
Can we change this  to demoapi since we need to deploy the changes?

Reply:
This PR is for Yale branch, just for history in case we need it. Demo is in my repo at https://github.com/pavelkrolevets/openx-frontend

Reply:
One sec Ill change URL